src/uncrustify.cpp

```diff
@@ -234,8 +235,8 @@ void usage_error(const char *msg)
 static void tease()
 {
    fprintf(stdout,
-           "There are currently %zu options and minimal documentation.\n"
```

Comment:
...this is *solely* because Xenial has a broken mingw compiler??

Reply:
I don't think